package.json

```diff
@@ -141,7 +141,7 @@
     "parse-dashboard": "./bin/parse-dashboard"
   },
   "engines": {
-    "node": ">=12.0.0 <16.0.0"
+    "node": ">=12.13.0"
```

Comment:
This is a breaking change that would require a dashboard release 4.0.

Is there any way to avoid the 12.13.0 bump?

Reply:
I’m pretty sure I bumped it because it was causing the tests to fail

Reply:
Found it, `sass-loader requires at least node 12.13.0 (>= 12.13.0)`

https://github.com/parse-community/parse-dashboard/runs/3698734537?check_suite_focus=true